bff-service/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import express from 'express';
+import dotenv from 'dotenv';
+import { AxiosRequestConfig, Method } from 'axios';
+dotenv.config();
+
+import request from './request';
+
+const app = express();
+const PORT = process.env.PORT || 8080;
+
+app.use(express.json());
+
+app.all('*', async (req, res) => {
+    console.log('OriginalUrl: ', req.originalUrl);
+    console.log('Method: ', req.method);
+    console.log('Body: ', req.body);
+
+    const recipient = req.originalUrl.split('/')[1];
+    console.log('Recipient: ', recipient);
+
+    const recipientUrl = process.env[recipient];
+    console.log('RecipientUrl: ', recipientUrl);
+
+    if (recipientUrl) {
+        const axiosConfig: AxiosRequestConfig = {
+            method: req.method as Method,
+            url: `${recipientUrl}${req.originalUrl}`,
+            ...(Object.keys(req.body || {}).length > 0 && { data: req.body }),
+        };
+
+        console.log('axiosConfig: ', axiosConfig);
+
+        try {
+            const result = await request(axiosConfig);
+            res.json(result);
```

Comment:
You missed sending status here

Reply:
As I understand, here we need a status code of 200, that is code by default (https://nodejs.org/api/http.html#http_response_statuscode)

Reply:
What's wrong with POST request? Product creation works correctly

Reply:
I mean, you will need a different status for successfull POST request.

Reply:
Yeah, you are right, thanks. Fogot about 201 status code for creating